project/client/static/scripts/script.js

```diff
@@ -52,8 +61,15 @@ function hide(elem) {
 
 /**
  * Show a given element by setting it's display attr.
+ * second argument allows specifying the display attribute
+ * @param {object}
+ * @param {string}
  */
-function show(elem) {
+function show(elem, display = false) {
```

Comment:
I am a fan of the Linux philosophy : Do one thing and do it well. I would have splittet this function into two functions. But because we are not going to add much more logic to our Javascript, this is also okay.